src/cap32.cpp

```diff
@@ -116,6 +116,11 @@ byte *membank_config[8][4];
 FILE *pfileObject;
 FILE *pfoPrinter;
 
+// Store shifted keys for correct remapping where shift is released before key release. 
+// Indexed by SDL_Scancode: https://wiki.libsdl.org/SDLScancodeLookup
```

Comment:
This documentation is for SDL 2. We're still using SDL 1.2.
(Migrating to SDL 2 would be nice but would also be a lot of work)